src/msw/textctrl.cpp

```diff
@@ -339,6 +339,8 @@ void wxTextCtrl::Init()
 {
 #if wxUSE_RICHEDIT
     m_verRichEdit = 0;
+    m_richBaseDPI = wxDefaultSize;
+    m_richUsedDPI = wxDefaultSize;
```

Comment:
Seems unnecessary, i.e. it looks like we could just leave them default-initialized, but harmless, of course.